local_deployment/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.9"
+services:
+  # run schematic APIs in a docker container with uWSGI and nginx
+  schematic-aws:
+    image: thomasvyu/schematic
+    entrypoint: python run_app.py
+    restart: always
+    environment:
+      APP_HOST: "0.0.0.0"
+      APP_PORT: "3001"
+      SCHEMATIC_CONFIG_CONTENT: ${SCHEMATIC_CONFIG_CONTENT}
+      SERVICE_ACCOUNT_CREDS: ${SERVICE_ACCOUNT_CREDS}
+      GE_HOME: app/great_expectations/
+    networks:
+      - schematic
+    ports: 
+      - "3001:3001"
```

Comment:
I think if this is removed, DCA could still execute schematic, but we just wouldn't be able to reach schematic.